JobConfig/common/geom_common_phase1.txt

```diff
@@ -80,7 +80,7 @@ double mu2e.detectorSystemZ0 = 10171.;   // mm  G4BL: (17730-7292=9801 mm)
 #include "Mu2eG4/geom/calorimeter_CsI.txt"
 
 //CRV counters
-#include "Mu2eG4/geom/crv_counters_v07.txt"
+#include "Mu2eG4/geom/crv_counters_v08.txt"
```

Comment:
We need to make sure this configuration propagates to Mu2eG4/geom/geom_common_current.txt, which is currently included by geom_common_phase1.txt

Reply:
It is included in Mu2eG4/geom/geom_common_current.txt via Mu2eG4/geom/geom_2021_PhaseI.txt.